shimmer/learn/outputs.md

```diff
@@ -0,0 +1,82 @@
+---
+description: 'Stardust introduces changes to the UTXO model, which allow for new output types, features, and unlock
+conditions.'
+image: /img/iota-wiki.png
+keywords:
+
+- explanation
+- shimmer
+- outputs
+- unlock conditions
+- features
+- output types
+
+---
+
+# Output Types and Features
+
+Stardust introduces [changes to the UTXO model](/introduction/develop/explanations/what_is_stardust/rethink_utxo) that
+make it possible for outputs novel [output types](#output-types),  [unlock conditions](#unlock-conditions) and [features](#features).
```

Comment:
Not sure what the intended sentence is. Probably something like:

```suggestion
Stardust introduces [changes to the UTXO model](/introduction/develop/explanations/what_is_stardust/rethink_utxo), introducing novel [output types](#output-types),  [unlock conditions](#unlock-conditions), and [features](#features).
```

Reply:
Could we include a small part of the explanation of outputs being linked to, so we don't necessarily need to navigate to another page to understand outputs at least at a basic level?

Reply:
the link is to the line directly below.

Reply:
Yeah but it doesn't really tell what outputs are, only what types of outputs there are. The concept of an output is not explained.

Reply:
But I'm fine merging this and opening a separate issue for this if you are ok with that?

Reply:
yup, that's the main issue here. Thanks!